src/main/java/com/icthh/xm/ms/configuration/repository/impl/MemoryConfigStorageExcludeConfigDecorator.java

```diff
@@ -0,0 +1,68 @@
+package com.icthh.xm.ms.configuration.repository.impl;
+
+import com.icthh.xm.commons.config.domain.Configuration;
+import com.icthh.xm.ms.configuration.config.ApplicationProperties;
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Delegate;
+import org.springframework.util.AntPathMatcher;
+
+import static java.util.Collections.emptySet;
+import static java.util.function.Predicate.not;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.collections.CollectionUtils.isEmpty;
+
+@RequiredArgsConstructor
+public class MemoryConfigStorageExcludeConfigDecorator implements MemoryConfigStorage {
+
+    @Delegate(excludes = UpdateConfig.class)
```

Comment:
is it safe to use experimental feature?

Reply:
in this case yes, we can replace to direct implementation every time by click "delombok" button in IDE